getting-started/cookbooks/webinar1-myapp/.kitchen.yml

```diff
@@ -0,0 +1,31 @@
+---
+driver:
+  name: vagrant
```

Comment:
Can you add a DEVELOPER.md file on the cookbook explaining how to test using Test Kitchen? (this is cool!)

Reply:
Done.